# GetDiagnostics.srv
# 获取诊断信息

# 请求 (空)
---
# 响应
bool success
string message
controller_ros/DiagnosticsV2 diagnostics
